server/bin/pbench-verify-backup-tarballs

```diff
@@ -262,11 +262,16 @@ def compare_entry_lists(list_one_obj, list_two_obj, report, logger):
 
 def sanity_check(s3_obj, logger):
     # make sure the S3 bucket exists
+    if s3_obj.bucket_name is None:
+        logger.warning("Bucket not defined in config file - S3 backup is disabled.")
```

Comment:
This warning is issued on every run, right?  Should we consider making it quieter?  (I'm not sure what the options are for that, but perhaps we could add something to the config file?)